# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/android/AccessibleWrap.cpp
# Commit: 786805a2cdcc
# Full Hash: 786805a2cdcc4cdab632a955dace417062d0087e
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r effe12f2120a -r 786805a2cdcc accessible/android/AccessibleWrap.cpp
--- a/accessible/android/AccessibleWrap.cpp	Tue Nov 28 22:03:20 2023 +0000
+++ b/accessible/android/AccessibleWrap.cpp	Tue Nov 28 22:03:21 2023 +0000
@@ -121,26 +121,6 @@
   return nullptr;
 }
 
-bool AccessibleWrap::PivotTo(int32_t aGranularity, bool aForward,
-                             bool aInclusive) {
-  Accessible* result = DoPivot(this, aGranularity, aForward, aInclusive);
-  if (result) {
-    MOZ_ASSERT(result->IsLocal());
-    // Dispatch a virtual cursor change event that will be turned into an
-    // android accessibility focused changed event in the parent.
-    PivotMoveReason reason = aForward ? nsIAccessiblePivot::REASON_NEXT
-                                      : nsIAccessiblePivot::REASON_PREV;
-    LocalAccessible* localResult = result->AsLocal();
-    RefPtr<AccEvent> event = new AccVCChangeEvent(
-        localResult->Document(), this, localResult, reason, eFromUserInput);
-    nsEventShell::FireEvent(event);
-
-    return true;
-  }
-
-  return false;
-}
-
 Accessible* AccessibleWrap::ExploreByTouch(Accessible* aAccessible, float aX,
                                            float aY) {
   Accessible* root;